packages/changed-elements-react/src/api/VersionCompareManager.ts

```diff
@@ -434,20 +434,27 @@ export class VersionCompareManager {
       if (this.options.wantedModelClasses) {
         wantedModelClasses = this.options.wantedModelClasses;
       }
-      let filteredChangeElements = changedElements.slice()
       if (this.ignoredElementIds !== undefined) {
-        filteredChangeElements = this._filterIgnoredElementsFromChangesets(changedElements);
+        await this.changedElementsManager.initialize(
+          this._currentIModel,
+          this._targetIModel,
+          this._filterIgnoredElementsFromChangesets(changedElements),
+          this.wantAllModels ? undefined : wantedModelClasses,
```

Comment:
I didn't want to shallow copy or have another ref pointing at the changeElements list. But it does not matter because the function being called to filter does not mutate the data being passed to it. I reverted the code and took off the slice.